email-client/src/main/java/uk/gov/hmcts/cmc/email/EmailService.java

```diff
@@ -50,7 +50,7 @@ public void sendEmail(String from, EmailData emailData) {
     }
 
     @Recover
-    public void logSendMessageWithAttachmentFailure(RuntimeException exception, EmailData emailData) {
+    public void logSendMessageWithAttachmentFailure(RuntimeException exception, String from, EmailData emailData) {
```

Comment:
is the argument change needed?

Reply:
It think yes, it is needed.
```
first parameter of type Throwable. 
The Throwable first argument is optional (but a method without it will only be called if no others match).
Subsequent arguments are populated from the argument list of the failed method in order.
```